sdk/batch/Microsoft.Azure.Batch/src/AzureBatch.ruleset

```diff
@@ -0,0 +1,121 @@
+    <Rule Id="CA1707" Action="None" />
+    <Rule Id="CA1708" Action="None" />
+    <Rule Id="CA1710" Action="None" />
+    <Rule Id="CA1711" Action="None" />
+    <Rule Id="CA1714" Action="None" />
+    <Rule Id="CA1715" Action="None" />
+    <Rule Id="CA1716" Action="None" />
+    <Rule Id="CA1717" Action="None" />
+    <Rule Id="CA1720" Action="None" />
+    <Rule Id="CA1721" Action="None" />
+    <Rule Id="CA1724" Action="None" />
+    <Rule Id="CA1725" Action="None" />
+    <Rule Id="CA1801" Action="None" />
+    <Rule Id="CA1802" Action="None" />
+    <Rule Id="CA1806" Action="None" />
+    <Rule Id="CA1812" Action="None" />
+    <Rule Id="CA1814" Action="None" />
+    <Rule Id="CA1815" Action="None" />
+    <Rule Id="CA1819" Action="None" />
+    <Rule Id="CA1822" Action="None" />
+    <Rule Id="CA1823" Action="None" />
+    <Rule Id="CA2007" Action="Error" />
+    <Rule Id="CA2119" Action="None" />
+    <Rule Id="CA2211" Action="None" />
+    <Rule Id="CA2214" Action="None" />
+    <Rule Id="CA2217" Action="None" />
+    <Rule Id="CA2219" Action="None" />
+    <Rule Id="CA2225" Action="None" />
+    <Rule Id="CA2226" Action="None" />
+    <Rule Id="CA2227" Action="None" />
+    <Rule Id="CA2231" Action="None" />
+    <Rule Id="CA2244" Action="None" />
+    <Rule Id="CA9999" Action="None" />
+  </Rules>
+  <Rules AnalyzerId="Microsoft.CodeQuality.CSharp.Analyzers" RuleNamespace="Microsoft.CodeQuality.CSharp.Analyzers">
+    <Rule Id="Async001" Action="None" />
+    <Rule Id="Async002" Action="None" />
+    <Rule Id="Async003" Action="None" />
+    <Rule Id="Async004" Action="None" />
+    <Rule Id="Async005" Action="None" />
+    <Rule Id="Async006" Action="None" />
+    <Rule Id="CA1001" Action="None" />
+    <Rule Id="CA1003" Action="None" />
+    <Rule Id="CA1019" Action="None" />
+    <Rule Id="CA1032" Action="None" />
+    <Rule Id="CA1065" Action="None" />
+    <Rule Id="CA1200" Action="None" />
+    <Rule Id="CA1500" Action="None" />
+    <Rule Id="CA1507" Action="None" />
+    <Rule Id="CA1726" Action="None" />
+    <Rule Id="CA1821" Action="None" />
+    <Rule Id="CA2001" Action="None" />
+    <Rule Id="CA2109" Action="None" />
+    <Rule Id="CA2200" Action="None" />
+    <Rule Id="CA2234" Action="None" />
+  </Rules>
+  <Rules AnalyzerId="Microsoft.CodeQuality.VisualBasic.Analyzers" RuleNamespace="Microsoft.CodeQuality.VisualBasic.Analyzers">
+    <Rule Id="Async001" Action="None" />
+    <Rule Id="Async002" Action="None" />
+    <Rule Id="Async003" Action="None" />
+    <Rule Id="Async004" Action="None" />
+    <Rule Id="Async005" Action="None" />
+    <Rule Id="Async006" Action="None" />
+    <Rule Id="CA1001" Action="None" />
+    <Rule Id="CA1003" Action="None" />
+    <Rule Id="CA1019" Action="None" />
+    <Rule Id="CA1032" Action="None" />
+    <Rule Id="CA1065" Action="None" />
+    <Rule Id="CA1200" Action="None" />
+    <Rule Id="CA1500" Action="None" />
+    <Rule Id="CA1507" Action="None" />
```

Comment:
?

Reply:
What's your question here?

Reply:
Why the new file

Reply:
It's to manage FxCop analyzer rules, which I added